### PR TITLE
Document template support in OperatorPolicy

### PR DESCRIPTION
**Pull request by JustinKuli**
Refs:
 - https://issues.redhat.com/browse/ACM-11399

### CODE REVIEW

File: governance/template_support_intro.adoc [hunk 1]
```diff
@@ -54,12 +54,12 @@ The equivalent call might use the following syntax: `{{hub "(lookup "v1" "Secret
...
 | The output of template functions are not stored in policy related resource objects.
 
 | Processing
 | Processing occurs at runtime on the hub cluster during propagation of replicated policies to clusters. Policies and the hub cluster templates within the policies are processed on the hub cluster only when templates are created or updated.
-| Processing occurs in the `ConfigurationPolicyController` on the managed cluster. Policies are processed periodically, which automatically updates the resolved object definition with data in the referenced resources.
+| Processing occurs on the managed cluster. ConfigurationPolicies are processed periodically, which automatically updates the resolved object definition with data in the referenced resources. OperatorPolicies automatically update whenever a referenced resource changes.
```
Comment (dockerymick) [inline governance/template_support_intro.adoc:62]:
```suggestion
| Processing occurs on the managed cluster. Configuration policies are processed periodically, which automatically updates the resolved object definition with data in the referenced resources. Operator policies automatically update whenever a referenced resource changes.
```

Comment (dockerymick) [inline governance/template_support_intro.adoc]:
```suggestion
| Processing occurs on the managed cluster. Configuration policies are processed periodically, which automatically updates the resolved object definition with data in the referenced resources. Operator policies automatically update whenever a referenced resource changes.
```

File: governance/policy_operator.adoc [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 The operator policy controller allows you to monitor and install Operator Lifecycle Manager (OLM) operators across your clusters. Use the operator policy controller to monitor the health of various pieces of the operator and to specify how you want to automatically handle updates to the operator. You can also distribute an operator policy to managed clusters by using the governance framework and adding the policy to the `policy-templates` field of a policy on the hub cluster.
 
+You can also use templated values within the `operatorGroup` and `subscription` fields of an operator policy. For more information, see _Template processing_.
```
Comment (dockerymick) [inline governance/policy_operator.adoc]:
```suggestion
You can also use template values within the `operatorGroup` and `subscription` fields of an operator policy. For more information, see _Template processing_.
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: governance/template_support_intro.adoc (hunk 1 above)
Comment (jc-berger) [reply governance/template_support_intro.adoc]: good suggestion,  go ahead and merge it.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by dockerymick — APPROVED** [review]
Thanks for reviewing @jc-berger and approving! @JustinKuli this looks good to me

**Comment by JustinKuli** [comment]
LGTM! Thanks for the edits :)

**Pull request merged**